test/es-module/test-esm-loader-hooks.mjs

```diff
@@ -553,4 +553,140 @@ describe('Loader hooks', { concurrency: true }, () => {
     assert.strictEqual(code, 0);
     assert.strictEqual(signal, null);
   });
+
+  it('should invoke `initialize` correctly', async () => {
+    const { code, signal, stdout, stderr } = await spawnPromisified(execPath, [
+      '--no-warnings',
+      '--experimental-loader',
+      fixtures.fileURL('/es-module-loaders/hooks-initialize.mjs'),
+      '--input-type=module',
+      '--eval',
+      `
+        import os from 'node:os';
+        import fs from 'node:fs';
```

Comment:
Note that FS gets mocked, so it's best to avoid this unless you want the mock.

Reply:
I can probably remove that line entirely tbh. As long as the initialize hook is triggered is all I care about in this test.